src/app/libs/managesieve.ui/importer/SieveImportUI.js

```diff
@@ -26,53 +26,53 @@
      * Shows the import account dialog.
      */
     async show() {
-      return await new Promise(async (resolve) => {
-        const dialog = await (new SieveTemplateLoader()).load("./importer/account.import.tpl");
-        dialog.find(".sieve-import-progress").hide();
-        $("#container").append(dialog);
+      await new Promise(async (resolve) => {
+        const dialog = await (new SieveTemplate()).load("./importer/account.import.tpl");
+        dialog.querySelector(".sieve-import-progress").style.display = "none";
```

Comment:
`getElementsByClassName` is [a little bit faster](https://jsperf.com/queryselector-vs-getelementbyclassname/89).